swift.html.markdown

```diff
@@ -147,6 +148,11 @@ for value in myArray {
     }
 }
 
+// for loop with indices (array)
```

Comment:
Did Swift 2.0 change this? With `enumerate` it changed, did this change as well?

Swift 2.0:
```
for (index, item) in myArray.enumerate() {
    print("value: \(item) is in position: \(index)")
}
```